content/hardware/upgrading.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Upgrading MuSHR v3 to v4"
+date: 2018-11-28T15:14:54+10:00
+intro_image: imgs/single-car.jpg
+featured: false
+draft: false
+active: true
+summary: Replace your NVIDIA Jetson Nano with the new Xavier NX
+weight: 3
+---
+
+This page provides instructions for replacing your MuSHR racecar's NVIDIA Jetson Nano with the upgraded, more powerful Xavier NX processor.
```

Comment:
I think we need a sentence here that says that the V4 is the same struturally but if you would like to replace the nano, see below. Do we need instructions (mainly software) for the other cameras? Do they need specific drivers?

Reply:
I added that sentence and I added a section on upgrading the camera hardware (installing the new front cover, etc.) No additional software setup is required as the D455 uses the same RealSense libraries (just updated versions).